lib/hobo_fields/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module HoboFields
-  VERSION = "3.0.0"
+  VERSION = "3.1.0"
```

Comment:
I wonder about possibly going back to a 2.x version since this is now compatible with Rails 4 again? In our world, there aren't any breaking changes here, right? But to other organizations--who might have used the custom field types features which are removed here--it would be breaking.

Reply:
I think that since we went ahead and removed the rich formatters, we should leave this as a major version bump.  Untangling and getting those working is unnecessary for newer versions of rails, and going to be a mess of work with the shifting of responsibility that happened in rails 4 and 5.

Reply:
Ok.